examples/src/test/scala/com/gu/tableversions/examples/MultiPartitionTableLoaderSpec.scala

```diff
@@ -25,6 +27,8 @@ class MultiPartitionTableLoaderSpec extends FlatSpec with Matchers with SparkHiv
     implicit val tableVersions: TableVersions[IO] = InMemoryTableVersions[IO].unsafeRunSync()
     implicit val metastore: Metastore[IO] = new SparkHiveMetastore[IO]()
 
+    spark.sparkContext.hadoopConfiguration.set("fs.versioned.baseFS", "file")
```

Comment:
Maybe set this in the `TableLoader` code so you don't have to repeat it in each test.